packages/core/src/routes/connector.ts

```diff
@@ -137,7 +137,7 @@ export default function connectorRoutes<T extends AuthedRouter>(router: T) {
         await validateConfig(body.config);
       }
 
-      const connector = await updateConnector({ set: body, where: { id } });
+      const connector = await updateConnector({ set: body, where: { id }, fullyReplace: true });
```

Comment:
Seems that there is no UT for `PATCH /connectors/:id` for now?

Reply:
`fullyReplace` should always be `true` when updating `connector.config` and should not be user-configurable.